service/src/main/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellBaseResourcesFactory.java

```diff
@@ -186,18 +188,14 @@ public Deployable definition(DefinitionContext definitionContext) {
                   nameGenerator.nextName(ResourceNameGenerator.MAX_POSTGRESQL_SERVER_NAME_LENGTH))
               .withRegion(resourceGroup.region())
               .withExistingResourceGroup(resourceGroup.name())
-              .withProperties(
```

Comment:
note that `CromwellBaseResourcesFactory` does not work anymore. I did not want to take the time to update it for flexible servers.